src/dmd/root/filename.d

```diff
@@ -793,6 +793,7 @@ version(Windows)
      */
     private int _mkdir(const(char)* path) nothrow
     {
+        import core.sys.windows.winbase: CreateDirectoryW;
```

Comment:
Why is this necessary?

Reply:
It isn't, but I copied the code around and it failed to compile because of the import. It was one my one concession to "do only what you must". I don't mind deleting it.